modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/js/EditKBArticle.js

```diff
@@ -52,44 +52,54 @@ export default function EditKBArticle({
 		event.currentTarget.dataset.customUrl = urlTitleInput.value !== '';
 	};
 
+	const scheduleItemOnClick = () => {
+		openScheduleModal(Liferay.Language.get('schedule-publication'));
+	};
+
+	const scheduledButtonOnClick = () => {
+		openScheduleModal(Liferay.Language.get('edit-scheduled-publication'));
+	};
```

Comment:
Is this schedule vs scheduled difference on purpose? It looks a bit odd.

Reply:
Yeah it looks strange but was on purpose. The "schedule" modal is for both published or scheduled articles. But if it is scheduled, then the button "scheduled" appears with "Edit Scheduled Publication" title. Makes sense?

Reply:
Sounds good